ispyb/__init__.py

```diff
@@ -47,6 +47,24 @@ def open(configuration_file):
     return conn
 
 
+def sqlalchemy_session(configuration_file):
```

Comment:
I'm wondering whether this should be a `def session(...)` in `ispyb/sqlalchemy/__init__.py`.
Further, this should probably take `configuration=None` and then
```python
configuration = configuration or os.getenv("ISPYB_CREDENTIALS")
```
but appreciate opinions may vary on both accounts